.drone.star

```diff
@@ -6,7 +6,7 @@ def main(ctx):
             "tarball": "https://download.owncloud.org/community/owncloud-complete-20210720.tar.bz2",
             "tarball_sha": "d9a5a61b3ac4e6e34302b4d98c9aabc63009662cd5142d990fda87d890209aa6",
             "ldap": "https://github.com/owncloud/user_ldap/releases/download/v0.15.4/user_ldap-0.15.4.tar.gz",
-            "ldap_sha": "8246d1fb3fc9e6586a8a5951ba0adb103b1cc1eb96efdba9791a1d9fead6cace",
+            "ldap_sha": "84412529b941723ef9523fe3c6d7c43cf2e44ab2522a9a9933966228f96122de",
```

Comment:
why again a new checksum? does it need a rebase?

Reply:
Drone cannot succeed, when this has the old checksum. 
I try to avoid to do rebase myself, I most likely break things beyond repair, when I try.